pfsspy/__init__.py

```diff
@@ -512,6 +513,74 @@ def _common_b(self):
         self._common_b_cache = br, bs, bp, Sbr, Sbs, Sbp
         return self._common_b_cache
 
+    def get_Bvec(self,skycoord,out_type="cartesian") :
+        """
```

Comment:
This needs a sentence or two at the top of the docstring to explain what it does.

Reply:
Added a short and longer description and it has propagated to the docs build